internal/handlers/common/distinct.go

```diff
@@ -43,6 +43,7 @@ type DistinctParams struct {
 	Collation *types.Document `ferretdb:"collation,unimplemented"`
 
 	ReadConcern *types.Document `ferretdb:"readConcern,ignored"`
+	LSID        any             `ferretdb:"lsid,ignored"`
```

Comment:
As far as I can see, I don't spot them as command field in the documentation. See https://www.mongodb.com/docs/manual/reference/command/distinct/.

If we don't want to make the same mistake of forgetting to add lsid, what should we check? :thinking:

Reply:
The driver will attach an implicit session ID to every command if the server supports sessions and we support them as far as the client is concerned because we return a value for `logicalSessionTimeoutMinutes`.

See [here](https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#sending-the-session-id-to-the-server-on-all-commands).

> When connected to a server that supports sessions a driver MUST append the session ID to every command it sends to the server (with the exceptions noted in the following section). It does this by adding a top level lsid field to the command sent to the server. A driver MUST do this without modifying any data supplied by the application

> A driver can determine whether a connection supports sessions by checking whether the logicalSessionTimeoutMinutes property of the establishing handshake response has a value or not. If it has a value, sessions are supported.

Is there something we could do better here to avoid parsing errors?

Reply:
Oh I see :eyes:, then we should add ignore `lsid` for every command right?

For example this one has ignore which is good https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/pg/msg_aggregate.go#L54

But just to name a few, below ones don't have ignore on them. And there are more like that.
https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/pg/msg_collstats.go
https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/pg/msg_count.go'